shared/src/ledger/storage/merkle_tree.rs

```diff
@@ -617,5 +608,67 @@ mod test {
         }
         // Check the base root
         assert_eq!(sub_root, tree.root().0);
+
+        // Delete the pos key
+        tree.delete(&pos_key).unwrap();
+        assert!(tree.has_key(&ibc_key).unwrap());
+        assert!(!tree.has_key(&pos_key).unwrap());
```

Comment:
These assertions are not needed for the test. Deletion has been tested in another unit test.

Reply:
Fair point.